spec/e2e/active_model_spec.rb

```diff
@@ -436,6 +436,27 @@ def true_results?(node, verb)
       expect(person.name).to eq 'Wilson'
     end
 
+    it 'can increment an attribute' do
+      person = Person.create(name: 'andreas', age: 21)
+      person.increment(:age)
+      expect(person.age).to eq(22)
```

Comment:
It was kind a way to check check that `increment` is not saving, `increment!` is saving.
What about something like:

```ruby
    it 'can increment an attribute' do
      person = Person.create(name: 'andreas', age: 21)
      expect { person.increment(:age) }.to change { person.age }.from(21).to(22)
    end

    it 'can increment an attribute and save' do
      person = Person.create(name: 'andreas', age: 21)
      expect { person.increment!(:age) }.to change { person.age }.from(21).to(22)
      expect(person).not_to be_changed
    end
```